proto/internal/temporal/server/api/persistence/v1/tasks.proto

```diff
@@ -50,6 +50,9 @@ message TaskInfo {
     // TaskVersionDirective, which is unversioned.)
     // Deprecated. No need to store this for new versioning [cleanup-old-wv]
     temporal.server.api.taskqueue.v1.TaskVersionDirective version_directive = 8;
+
+    // Stamp field allows to differentiate between different instances of the same task
```

Comment:
```suggestion
    // A version marker that is used to drop tasks that are no longer relevant to their corresponding state machine. 
```

Reply:
The name stamp is too generic for a task that definition that is fairly broad. I'd call this `conflict_token` (here and everywhere else).

Reply:
Will discuss offline, so far I like `stamp` better. I also change the description.

Reply:
keeping this open in case we come with a better name

Reply:
I would at least take the suggested documentation change.

Reply:
I added a more detailed comments .